pkg/network/ebpf/c/protocols/grpc/defs.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __GRPC_DEFS_H
+#define __GRPC_DEFS_H
+
+typedef enum {
+  GRPC_STATUS_UNKNOWN,
+  GRPC_STATUS_GRPC,
+  GRPC_STATUS_NOT_GRPC,
```

Comment:
Originally, the `STATUS` part wasn't there, but I didn't find `GRPC_GRPC` and `GRPC_NOT_GRPC` to be very readable either. Do you have an idea for a better naming scheme? I specifically wanted to avoid having a global `UNKNOWN` name, which would be prone to name conflicts.